Open Judge System/Services/OJS.Services/Business/Contests/Models/JudgeWorkLoadResults.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace OJS.Services.Business.Contests.Models
+{
+    public class JudgeLoadResults
+    {
+        //Judge work
+        public int Submissions { get; set; }
+
+        public int TotalWork  => (int)Math.Ceiling((this.Submissions * 2) / 60.0);
```

Comment:
Same here. What is `TotalWork`? Is it for all workers, or total per worker, also is it seconds, minutes, etc.. If the property name is not self-descriptive enough, just add a \<summary> explaining it.